Makefile

```diff
@@ -25,6 +25,7 @@ JSFILES=$(shell find . -wholename './node_modules*' -prune \
 	| sort | sed -e 's/^\.\///' \
 	| grep -Ev \
 		-e '^app/assets/javascripts/d3(\.min)?\.js$$' \
+		-e '^app/assets/javascripts/react-[0-9]+\.[0-9]+\.[0-9]+(\.min)*\.js$$' \
```

Comment:
It doesn't but I could make it a little different if you'd prefer

`^app/assets/javascripts/react-[0-9]+(\.[0-9]+){2}(\.min)*\.js$$`